web-app/js/portal/details/SpatialConstraintDisplayPanel.js

```diff
@@ -62,7 +64,9 @@ Portal.details.SpatialConstraintDisplayPanel = Ext.extend(Ext.Panel, {
         if (type == Portal.ui.openlayers.SpatialConstraintType.BOUNDING_BOX) {
             card = this.boxDisplayPanel;
         }
-
+        else if (type == Portal.ui.openlayers.SpatialConstraintType.POINT) {
```

Comment:
I would say this is cleaner, but this is also an opportunity to refactor using an OO approach